interface-definitions/firewall.xml.in

```diff
@@ -100,6 +100,30 @@
               #include <include/generic-description.xml.i>
             </children>
           </tagNode>
+          <tagNode name="domain-group">
+            <properties>
+              <help>Firewall domain-group</help>
+              <constraint>
+                <regex>[a-zA-Z0-9][\w\-\.]*</regex>
```

Comment:
Would recommend removing digit from first character group, without a prefix on the nftables set name a starting digit would cause nftables not to parse the file properly.

Faulty example: `set firewall group domain-group 1Test ...`

Reply:
Done